packages/interaction-kit/src/definitions/permissions.ts

```diff
@@ -37,4 +37,5 @@ export enum PermissionFlags {
 	MANAGE_THREADS = 1 << 34,
 	USE_PUBLIC_THREADS = 1 << 35,
 	USE_PRIVATE_THREADS = 1 << 36,
+	MODERATE_MEMBERS = 1 << 40,
```

Comment:
I think we need to move these to BigInts, right?

Reply:
Not sure if this is the correct way to do it, at least this is how it's done in [discord-api-types](https://github.com/discordjs/discord-api-types/blob/main/payloads/v9/permissions.ts#L14-L56).